configs/sandpack-contents/component-theming/divider.js

```diff
@@ -0,0 +1,89 @@
+module.exports = {
+    App: `import { Box, SimpleGrid, IconButton, Center, Divider, useColorMode } from "@chakra-ui/react";
+import { FaMoon, FaSun } from "react-icons/fa";
+export default function App() {
+    const { toggleColorMode, colorMode } = useColorMode();
+    return (
+        <Box position="relative" h="100vh">
+            <SimpleGrid gap={12} p={12} columns={2}>
+              <Divider variant="thick" />
+              <Center height="50px">
+                <Divider orientation="vertical" variant="thick" />
+              </Center>
+              <Divider variant="brand" />
+              <Center height="50px">
+                <Divider orientation="vertical" variant="brand" />
+              </Center>
+              <Divider size="xl" />
+              <Center height="50px">
+                <Divider orientation="vertical" size="xl" />
+              </Center>
+            </SimpleGrid>
+            <IconButton
+                rounded="full"
```

Comment:
The `IconButton` requires an `aria-label` prop. Put this as the first prop: `aria-label="toggle theme"`